Scripts/Mac/PrepFunctionalTests.sh

```diff
@@ -33,15 +33,13 @@ fi
 
```

Comment:
Unrelated to the current changes, but line 10 above fails with a non-informative error if you run this script before you have built. Might want to just check that the package exists before trying to attach it.

```
Saeeds-MBP:src sanoursa$ Scripts/Mac/PrepFunctionalTests.sh 
hdiutil: attach failed - No such file or directory
```

Reply:
I'll clean that up while I'm here. Thanks!